config/etfuturum/functions.cfg

```diff
@@ -0,0 +1,278 @@
+    B:enablePlayerSkinOverlay=false
+
+    # Enable subtitles [default: false]
+    B:enableSubtitles=false
+
+    # Allow non-opaque armour [default: true]
+    B:enableTransparentAmour=true
+
+    # Render beds with a 3D inventory model instead of a 2D sprite. [default: true]
+    B:inventoryBedModels=true
+
+    # The maximum amount of items a Shulker box can display on its tooltip. When the box has more stacks inside than this number, the rest of the stacks are displayed as "And x more...". Set to 0 to disable Shulker Box tooltips. [range: 0 ~ 127, default: 5]
+    I:shulkerBoxTooltipLines=5
+
+    # Regex of subtitles to blacklist [default: ^(dig\.*)]
+    S:subtitleBlacklist=^(dig\.*)
+}
+
+
+##########################################################################################################
+# settings
+#--------------------------------------------------------------------------------------------------------#
+# Settings for Et Futurum content.
+##########################################################################################################
+
+settings {
+    # The data watcher flag for the Elytra, used to sync the elytra animation with other players. In vanilla the max value is 7, players use 0-4, so you can set this to 6 or 7 by default. ASJCore increases the max value to 31.
+    # Do not change this value if you don't need to, or do not know what you're doing. [range: 0 ~ 31, default: 7]
+    I:elytraDataWatcherFlag=7
+
+    # Seeks all available smeltable ores, metals, etc (using OreDict tags like "ore", "cluster", "ingot", etc) from the furnace and adds them to the Blast Furnace, if it's off it will only smelt things specified from CraftTweaker. [default: true]
+    B:enableAutoAddBlastFurnace=true
+
+    # Seeks compostable items like plants and leaves, and automatically adds them to the composter. If this is off it will only compost things specified from CraftTweaker and nothing by default. [default: true]
+    B:enableAutoAddComposter=true
+
+    # Seeks all available edible foods from the furnace and adds them to the smoker, if it's off it will only smelt things specified from CraftTweaker. [default: true]
+    B:enableAutoAddSmoker=true
+
+    # Set to true to disable the fireproof item entity Netherite/ancient debris etc uses [default: false]
+    B:enableNetheriteFlammable=false
+
+    # Recipe for prismarine if you want it without the temples, or want it craftable alongside temples. [default: true]
+    B:enablePrismarineRecipes=true
+
+    # Recipe for totems since there's no other way to get them currently. [default: false]
+    B:enableRecipeForTotem=false
+
+    # OreDictionary values for ore blocks that should drop extra items (2-3) instead of the usual one, before fortune. [default: [oreCopper], [oreTin]]
+    S:extraDropRawOres <
+        oreCopper
+        oreTin
```

Comment:
this should be empty. we do rawores in gt

Reply:
Done.

Reply:
I should probably have clarified this affects EFR raw ores only so touching it doesn't do anything since you already disabled those.